vsintegration/src/FSharp.Editor/Formatting/EditorFormattingService.fs

```diff
@@ -167,14 +167,17 @@ type internal FSharpEditorFormattingService
             return textChanges |> Option.defaultValue Seq.empty |> toIList
         }
         
-    interface IFSharpEditorFormattingService with
```

Comment:
I need to review that one, please don't merge it.

Reply:
It's fine just a smart indent change.